/bin/sh: -c: line 0: unexpected EOF while looking for matching `"'
/bin/sh: -c: line 1: syntax error: unexpected end of file
Traceback (most recent call last):
  File "/gpfs/home/asun/miniforge3/envs/yap/bin/yap", line 8, in <module>
    sys.exit(main())
  File "/gpfs/home/asun/miniforge3/envs/yap/lib/python3.8/site-packages/cemba_data/__main__.py", line 864, in main
    func(**args_vars)
  File "/gpfs/home/asun/miniforge3/envs/yap/lib/python3.8/site-packages/cemba_data/demultiplex/demultiplex.py", line 446, in demultiplex_pipeline
    subprocess.run(f'cp {config_path} {new_config_path}',
  File "/gpfs/home/asun/miniforge3/envs/yap/lib/python3.8/subprocess.py", line 516, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command 'cp "/gpfs/home/asun/jin_lab/yap/pipeline2_yap_iseq0328/mct_config.ini" /gpfs/group/jin/asun/yap/pipeline2_yap_iseq0328/"/gpfs/home/asun/jin_lab/yap/pipeline2_yap_iseq0328/yap_demultiplex"/mapping_config.ini"' returned non-zero exit status 1.
